test/__mocks__/text.mock.ts

```diff
@@ -127,4 +127,8 @@ export const CHANGE_LINK_INDIVIDUAL_BO_IS_ON_SANCTIONS_LIST = CHANGE_LINK_INDIVI
 
 // Update journey
 export const OVERSEAS_ENTITY_QUERY_PAGE_TITLE = "What is the OE number?";
-export const OE_NUMBER_FIELD_POPULATED = "value=\"OE123456\"";
+export const CONFIRM_OVERSEAS_ENTITY_PAGE_TITLE = "Confirm this is the registered overeseas entity";
+export const UPDATE_OVERSEAS_ENTITY_PAGE_TITLE = "Confirm or update overseas entity details";
+export const CONFIRM_AND_CONTINUE_BUTTON_TEXT = "Confirm and continue";
+export const UPDATE_OVERSEAS_ENTITY_TITLE = "Update overseas entity";
+export const OE_NUMBER_FIELD_POPULATED = "value=\"12345678\"";
```

Comment:
As the validation ticket has been merged to main, the OE number will have to be OE123456